File: 100-Days-Series/Day 88/PerfectCubeNumber.cpp
#include<iostream>
using namespace std;
// Function to find Perfect Cube Number between 1 to N.
void getFindPerfectCubeNumber(int n){
    for(int i=1;i<=n;i++){
        int res=i*i*i;
        if(res>n){
            break;
        }
        cout<<res<<" ";
    }
    cout<<endl;
}
int main(){
    int n;
    cout<<"\nEnter the value of N :";
    cin>>n;
    cout<<"\nPerfect Cube Number between 1 to "<<n<<endl;
    getFindPerfectCubeNumber(n);
}

File: 100-Days-Series/Day 88/PerfectCubeNumber.py

# Function to find Perfect Cube Number between 1 to N
def get_find_perfect_cube_number(n):
    for i in range(1,n+1,1):
        res=i*i*i
        if res>n:
            break
        print(res,end=" ")


# Let's Start Here
n=int(input('Enter the value of N : '))
print('Perfect Cube Number between 1 to ',n,' :')
get_find_perfect_cube_number(n)